group.py

```diff
@@ -1,5 +1,32 @@
-"""An example of how to represent a group of acquaintances in Python."""
-
-# Your code to go here...
-
-my_group =
+friends = {
```

Comment:
This code looks very clean - I really like the formatting and spacing that you've used. It makes it a lot easier than having everything on one line.